pyuptimerobot/uptimerobot.py

```diff
@@ -22,5 +22,7 @@ async def async_get_account_details(self, **kwargs) -> UptimeRobotApiResponse:
         """Get account details from API."""
 
     @api_request("/editMonitor")
-    async def async_edit_monitor(self, **kwargs) -> UptimeRobotApiResponse:
+    async def async_edit_monitor(
+        self, monitor_id: int, **kwargs
```

Comment:
```suggestion
        self,
        id: int,
        **kwargs,
```
`monitor_id` and `id` is not the same